### DISCUSSION TITLE
Deploying Hydrogen store -> After build got 'could not resolve' error

### DISCUSSION THREAD

**Discussion by DennisKraaijeveld** [vercel/community#881] [Help] [answered]
Hi! I am developing a new Hydrogen/Sanity store and I did want to test out deploying this to Vercel. After build is completed, the deployment returns the following ERROR:

```
Build Completed in /vercel/output [1m]
--
14:31:23.310 | Build failed with 1 error:
14:31:23.311 | vc-file-system:index.js:789:19511: ERROR: Could not resolve "scriptjs"
```

What could be the problem? I tried looking into scriptjs but this is a dependency of a React package.

**Comment by amyegan** [comment, marked as answer]
Hi @DennisKraaijeveld! You should check that your project is using the correct [Build & Development Settings](https://vercel.com/docs/concepts/deployments/configure-a-build#build-and-development-settings) for your framework (either an optional preset or custom settings). That is sometimes the source of package installation problems. It could also be a bug in a package you have installed. Are you able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can get a better view of the problem?
